lib/src/builders/method.dart

```diff
@@ -186,7 +186,18 @@ abstract class ConstructorBuilder
         HasStatements,
         ValidClassMember {
   /// Create a new [ConstructorBuilder], optionally with a [name].
-  factory ConstructorBuilder([String name]) = _NormalConstructorBuilder;
+  factory ConstructorBuilder({
+    String name,
+    String superName,
+    List<ExpressionBuilder> invokeSuper,
+  }) = _NormalConstructorBuilder;
+
+  /// Adds a constructor initializer.
```

Comment:
[nit] 'constructor initializer' reads a bit strange to me... how about

/// Adds a field initializer to this constructor.